### DISCUSSION TITLE
Voice on android auto

### DISCUSSION THREAD

**Discussion by J3rr1e** [osmandapp/OsmAnd#18829] [Q&A]
I have OsmAnd navigation directly loaded into my android radio. I can't get the voice direction to work even though the navigation works perfectly. I've tried multiple adjustments without success. Does OsmAnd voice not work when the app is directly downloaded into the radio. I'd prefer to not have to use my phone as a connection to the radio. Any suggestions would be appreciated.

**Comment by DmitryAlexei** [comment]
Have you tried ideas from troubleshooting? https://osmand.net/docs/user/troubleshooting/navigation#voice-navigation

**Comment by J3rr1e** [comment]
Thank you for your input.I have looked through the troubleshooting.
Although I finally got the voice to work by going in and the voice to use
the systems settings. The brought up voice direction but I believe it's
using recorded not tts. I'm not finding anything referencing a tts engine.
I'm still working on that.
